File: Backend/proyecto/src/main/java/com/admindb/proyecto/modelo/tabla/Tabla.java
package com.admindb.proyecto.modelo.tabla;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

import lombok.Getter;
import lombok.Setter;



@Getter
@Setter
@Entity
@Table(name = "dba_tables")
public class Tabla {
    
    @Column(name = "owner")
    private String propietario;

    @Id
    @Column(name = "table_name")
    private String nombre;
}

File: Frontend/frontend-adminbd/src/app/common/tabla.ts
import { Restriccion } from "./restriccion";

export class Tabla {

    propietario: string;

    nombre: string;

    comentarios: string[];

    restricciones: string[];

    constructor(propietario: string, nombre: string,comentarios:string[], restricciones: string[]) {
        this.propietario = propietario;
        this.nombre = nombre;
        this.comentarios = comentarios;
        this.restricciones = restricciones;
     }
}

